lib/core/test/integration/Test/Integration/Framework/DSL.hs

```diff
@@ -711,6 +714,22 @@ tearDown ctx = do
          Left e -> error $ "deleteAllWallets: Cannot return wallets: " <> show e
          Right s -> s
 
+-- | Wait a booting wallet server to has started. Wait up to 30s or fail.
```

Comment:
_Wait a booting wallet server ~to~ has started_ (or _to ~has~ start~ed~_)

Reply:
Aaaah. Damn. I clicked merge too fast :man_facepalming: 
I'll fix in another PR.